citi-onboarding/Delta-odonto
# Feature/front who we are

## What i did

- 'Quem somos' section

## How to test

- On one terminal you should type the following commands to access the client folder and run the Keystone example
`cd client`
`npm start` or `yarn start`

- So you should see the section 'quem somos'. In the right you should see a photo of member, and in the left you should be the text and under it the icons as a table 2x3

- In the mobile format you shoud see the section without the photo and only text and icons like the desktop model
